PythonVisualizations/Stack.py

```diff
@@ -0,0 +1,235 @@
+import time
+from tkinter import *
+try:
+    from drawable import *
+    from VisualizationApp import *
+except ModuleNotFoundError:
+    from .drawable import *
+    from .VisualizationApp import *
+
+CELL_SIZE = 40
+CELL_BORDER = 2
+CELL_BORDER_COLOR = 'black'
+STACK_X0 = 300
+STACK_Y0 =  400
+FONT_SIZE = 20
+VALUE_FONT = ('Helvetica', FONT_SIZE)
+VALUE_COLOR = 'black'
+
```

Comment:
Please merge `origin/master` into this issue branch to change these global constants into class attributes.  The references in the methods to these constants will then need to be prefixed with `self.`